iris-mpc-common/src/galois_engine.rs

```diff
@@ -364,7 +414,7 @@ pub mod degree4 {
         let store_iris_shares = code_share.clone();
         let store_mask_shares = mask_share.clone();
 
-        // With rotations for in-memory database.
+        // With rotations for in-memory database. // TODO: is this dead code?
```

Comment:
It is needed for the batch cross-check IIRC.

Reply:
ah right!